contrib/builder/rpm/generate.sh

```diff
@@ -121,7 +121,7 @@ for version in "${versions[@]}"; do
 			&& set -x \
 			&& yum install -y $buildDeps \
 			&& export SECCOMP_PATH=$(mktemp -d) \
-			&& git clone -b "$SECCOMP_VERSION" --depth 1 https://github.com/seccomp/libseccomp.git "$SECCOMP_PATH" \
+			&& git clone -b "v${SECCOMP_VERSION}" --depth 1 https://github.com/seccomp/libseccomp.git "$SECCOMP_PATH" \
```

Comment:
I think both RPM and deb builders need to re-sync to https://github.com/docker/docker/blob/4e9ee2a239747fb19e26818cf8ed95d8bbfc3e27/Dockerfile#L142-L143 :innocent: